modules/client/src/index.ts

```diff
@@ -5,3 +5,5 @@ export * from "./client-common";
 export * from "./multisig";
 export * from "./types";
 export * from "./constants";
+// TODO review
+export * from "@aragon/sdk-client-common"
```

Comment:
@brickpop Not really sure about this one, Idont want peoble to download the client common in order to be able to use things like `Context` or `ContextParams`, this solves the problem but I'm not sure if it is the best solution, let me know what do you think

Reply:
Why would they need to be using sdk-client-common through sdk-client?

If they want to build a plugin, they should import sdk-client-common directly.
If there are any interfaces we want to expose here, then let's export them explicitly, and not through an `export *`